### DISCUSSION TITLE
Failed dependancy script.skinvariables version 1.0.0

### DISCUSSION THREAD

**Discussion by Donderstraal** [jurialmunkey/skin.arctic.horizon.2#1021] [Discussion]
After installing your repository manually in Kodi 21 I get the dependancy error on script.skinvariables.
If I try to install manually from your repo it offers version 2.1.10 and gives the error script.module.jurialmunkey version 0.1.17.

Is there something I'm missing here? Never had this problem with your old skins or repos.

**Comment by sgeg06** [comment]
Hello had the same issue on TV…
Instead to install directly the dependencies from the repository you must update it from the dependency management and select  jurialmunkey repository to upgrade
Edit: same for the tmdbhelper version

**Comment by Donderstraal** [comment]
Well I tried a few times to no avail... I give up and might revert back to the previous Kodi version.
Somehow it just can't be done.
Is there someone who can supply a .zip with the failing addons being needed to install them manually?
